Flat/src/JetsMods.cc

```diff
@@ -574,7 +574,7 @@ void HbbSystemMod::do_execute()
   sort(btagsorted.begin(), btagsorted.end(),
        analysis.useCMVA ?
         [](const JetWrapper *x, const JetWrapper *y) { return x->base->cmva > y->base->cmva; } :
-        [](const JetWrapper *x, const JetWrapper *y) { return x->base->csv > y->base->csv; }
+        [](const JetWrapper *x, const JetWrapper *y) { return x->base->deepCSVb > y->base->deepCSVb; }
```

Comment:
I don't think you want this - you're turning on deepCSV for 2016 too. Careful with these hacks.

Reply:
Even in 2016? Okay, if so, I'll merge. (Please do confirm)

Reply:
yes